### ISSUE TITLE
timeseries module missing

### ISSUE THREAD

**Issue by BriceChivu** [feature-engine/feature_engine#467]
Hi there,

It seems that the module 'timeseries' is missing when installing feature_engine library. 
I tried both pip and conda installation.

![image](https://user-images.githubusercontent.com/48819468/172743228-a32296f9-e5eb-49db-86c5-d5f734943903.png)

Any idea why it is so?

Thanks in advance

Brice

**Comment by solegalli** [comment]
which version of feature engine are you using?

from the notebook you can check as follows:

```
import feature_engine
feature_engine.__version__
```

time series is available only from version 1.3 onward

**Comment by BriceChivu** [comment]
Hi,
Thanks for your quick reply.
The version is 1.3.0
![image](https://user-images.githubusercontent.com/48819468/172785152-e808473a-bb9b-4351-9ff5-962f6a7608da.png)

**Comment by solegalli** [comment]
OK, thank you.

What is the error message? or the question about? can you import WindowFeatures? or does that raise an error? if so, could you copy and paste the error?

**Comment by BriceChivu** [comment]
The issue I'm facing is the missing timeseries module. Hence, I can't import WindowFeatures.
Here is the error message.
![image](https://user-images.githubusercontent.com/48819468/172786968-0b3f5893-0f17-4022-b485-fddbd79bb140.png)

**Comment by solegalli** [comment]
Yep, thanks for reporting. It seems we forgot an init file in the module. Fixing as we speak. Stay tuned.

**Comment by solegalli** [comment]
Hi @BriceChivu 

There is a new version now in Pypi, 1.4.0, that should resolve the issue. It would be great if you could try it out and let me know if it works.

The conda version will be available in a few hours.

Cheers
Sole

**Comment by BriceChivu** [comment]
Hello @solegalli 
I was able to use WindowFeatures after upgrading to 1.4.0
Thank you for that.

By the way, I have a follow-up question in the context of time-series forecasting. Using _WindowFeatures_, I wanted to use the target variable to create a rolling average feature. As expected, it gave NaN for the first row as there is no record before that, so far so good.
The problem arises when I want to use the transformed X_train to train a RandomForest model (or any other models that cannot deal with missing values). I tried using the method _DropMissingData_ but it drops NaN rows only for X, not y.

**So my question is**: Is there a way to also drop the row(s) in y when dropping missing data? Or are we constrained to only use models that can handle missing data when using the _WindowFeatures_ method to create a rolling average on the target?

Thanks in advance,

Cheers,
Brice

![image](https://user-images.githubusercontent.com/48819468/173195222-44204098-cffb-4137-a6dc-3ea1c0109c30.png)

**Comment by solegalli** [comment]
Hi @BriceChivu 

At the moment, you would have to remove missing data manually. I'd recomment creating a pipeline only for feature engineering. Then a step to remove nan in train set and target. Then train a model.

We are working on a forecasting pipeline in this PR #464 that would remove missing data from train and target automatically. Probably available later this year.

These are our first classes in the field of forecasting, so user feedback is very welcome. Thank you and keep it coming!

FYI
@KishManani

**Comment by BriceChivu** [comment]
Hello Soledad,

Noted on that. Appreciate your response.
Keep up the good work.

Cheers,
Brice
